pkgs/tools/backup/borg/default.nix

```diff
@@ -58,7 +58,7 @@ python3.pkgs.buildPythonApplication rec {
     HOME=$(mktemp -d) py.test --pyargs borg.testsuite
   '';
 
-  # 63 failures, needs pytest-benchmark
+  # 64 failures, needs pytest-benchmark
```

Comment:
We have `pytest-benchmark` since eb6e376753619cb1fcb57e3ef2075e836ec9dd3e (and it got bumped in the meantime as well). Can you take a look if we can run some of the tests?

Reply:
Lot's of `PermissionError` exceptions and it's also complaining about fuse not working.

https://gist.github.com/mweinelt/d5c1dbfc78cdbd02ee46a688f7f019f1

Reply:
This look like the test suite simply tries to use the current working directory (which is read-only), instead of setting up a temporary scratchdir.

You could try a `cd $(mktemp -d)` before invoking the tests, and if that works, ask upstream to use https://pytest.readthedocs.io/en/2.7.3/tmpdir.html instead of just assuming the folder is write-able.

Reply:
I think that is already happening here, no? https://github.com/NixOS/nixpkgs/pull/82052/files/dbff9b54795574a8bfbf7f679cdecb766aab9455#diff-2bc3b2ddae37125a5bc6c96304db7b08L58